src/lib/useKeyfileGenerator.js

```diff
@@ -32,9 +32,12 @@ export default function useKeyfileGenerator(manualNetworkSeed) {
   const [code, setCode] = useState(false);
 
   const _point = need.point(pointCursor);
-  const _details = need.details(getDetails(_point));
+  const _details = getDetails(_point);
```

Comment:
I think this should be `details` then. The `_` in this context generally indicates "we unwrapped it from `Maybe` already".